ui/src/shared/actions/me.ts

```diff
@@ -26,6 +28,18 @@ export const getMe = () => async dispatch => {
   try {
     const user = await client.users.me()
 
+    if (CLOUD) {
+      const orgID = getOrgIDFromUrl()
```

Comment:
this seems fragile and couples a view routing convention (that is currently changing with multi user and sample data work) with business logic that is meant to represent "this is my current organization". why isn't it coming back from the api?

Reply:
yeah this is pretty brittle

Reply:
so why not add it to the /me endpoint?